lib/virt_feature_test_base.pm

```diff
@@ -156,9 +162,10 @@ sub post_fail_hook {
 
     $self->{"stop_run"} = time();
     assert_script_run("history -w /root/commands_history");
-    virt_utils::stop_monitor_guest_console;
+    virt_utils::stop_monitor_guest_console() if (!(get_var("TEST", '') =~ /qam/) && (is_xen_host() || is_kvm_host()));
     #(caller(0))[3] can help pass calling subroutine name into called subroutine
     $self->junit_log_provision((caller(0))[3]) if get_var("VIRT_AUTOTEST");
+    collect_virt_system_logs();
```

Comment:
You do not add condition for this collect_virt_system_logs() this should be confined to qam use

Reply:
> 1. You do not add condition for this collect_virt_system_logs() this should be confined to qam use

@pdostal I think this is a valid point. Would you please add it in later PR?